jobs/awslogs/templates/bin/additional-logs.sh

```diff
@@ -17,3 +17,4 @@ do
   echo "initial_position = start_of_file"
   echo "log_group_name = $GROUP_NAME"
 done > /var/vcap/jobs/awslogs/conf.d/all-vcap-logs.conf
+monit restart awslogs
```

Comment:
Looking through docs: 

They service-restart the daemon themselves here: http://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/AgentReference.html 

The daemon is really just running the `aws logs push` command from the AWS cli tools. If there's a way to do it more elegantly, I haven't found it --